builtin_op_importers.cpp

```diff
@@ -1703,6 +1704,13 @@ DEFINE_BUILTIN_OP_IMPORTER(Slice) {
   for (size_t i = 0; i < axes.size(); ++i)
   {
     int axis = axes[i];
+    
+    // ignore slicing over batch dim
+    if (axis == 0)
```

Comment:
This case is already handled below in `TRT_CHECK(convert_axis(axis,nbDims)`, unneeded here.

Reply:
There are cases where onnx model does contain Slice op over batch dim. And the original code will exit with assertion or give incorrect result.  

My case:
![image](https://user-images.githubusercontent.com/17376949/55618549-60778780-57c9-11e9-9ad4-746f21200e72.png)

onnx exported from pytorch 1.01

Reply:
`convert_axis` currently sets the axis to `-1` if it was originally `0`, and then fails asserting that `axis >= 0`. Slicing over batch dimension should probably be allowed for slices such as `[:, :, h1:h2, w1:w2]`.